src/pages/home/sidebar/SidebarScreen/sidebarPropTypes.js

```diff
@@ -1,9 +1,6 @@
-import PropTypes from 'prop-types';
 import {windowDimensionsPropTypes} from '../../../../components/withWindowDimensions';
 
 const sidebarPropTypes = {
-    /** Callback when onLayout of sidebar is called */
-    onLayout: PropTypes.func,
```

Comment:
@adamgrzybowski could you kindly explain why you removed this prop type? we were not using this prop?

Reply:
cc @WoLewicki

Reply:
Yeah, iirc this value wasn't ever set so I thought it was better to just remove it than to stick to passing it, and add it on purpose if it will be used again.